sound/Music.java

```diff
@@ -203,21 +213,58 @@ public void playTune(Tune tune) {
         while (tuneIt.hasNext()) {
             System.out.println("[Line " + (line++) + "] ");
             chord = tuneIt.next();
-            playChord(chord,duration);
+            playChord(chord, duration);
         }
         silence();
     }
-    
+
     /**
      * Stop all tones coming from synthesizer.
      */
     public void silence() {
         Note note;
         int numNotes = noteHistory.size();
-        for (int i=0; i < numNotes; i++) {
+        for (int i = 0; i < numNotes; i++) {
             note = noteHistory.get(i);
-            channels[note.channel()].noteOff(note.pitch(),note.velocityOn());
+            channels[note.getChannel()].noteOff(note.getPitch(),
+                    note.getVelocityOn());
         }
     }
 
+    /**
+     *@return synthesizer interface
+     *        reference
+     */
+      Synthesizer getSynthesizer() {
+         return synth;
+     }
+
+     /**
+      *@return channels in this synth
+      */
+     MidiChannel[] getChannels() {
+        return channels;
+     }
+
+     /**
+      *@return Midi instrument number
+      */
+     int getInstrument() {
+        return instrument;
+     }
+
+     /**
+      *@return this notes velocity
+      */
+      int getVelocity() {
+         return velocity;
+      }
+
+     /**
+      *@return number of channels available
+      *        for this synth
+      */
+      int getNumChannels() {
+         return numChannels;
```

Comment:
All of these lines should be deleted since the information is not used outside the Music class.